src/widgets/_windows.scss

```diff
@@ -117,6 +117,23 @@ window:not(.popup):not(.menu) {
             text-shadow: none;
             -gtk-icon-shadow: none;
         }
+
+        pane-separator {
+            background-color: rgba(black, 0.0);
+            box-shadow: none;
+        }
+
+        separator {
+            background: none;
```

Comment:
Is there a reason to believe that `.titlebar separator` has a background set?

Reply:
Some apps use it to make dividers, and those for some reason have a grey background if left unstyled..